openshift/machine-config-operator
# removed the workerlatencyprofile status related code

**- What I did**
Removed the code related to the worker latency profile status updates from the config Node custom resource
**- How to verify it**
Update the config node CR with a relevant worker latency profile type and observe that there is no status update present in the config Node CR
**- Description for the changelog**

The worker latency profile status update will be written to the corresponding operator statuses such as KubeAPIServer Operator and the Kube Controller Manager status. Hence, it is not required to update the config Node CR status again.
Also, a piece of code has been added to generate events whenever there is an update happening on the config node object.
